javascript/javascript-core/intro-js/console-in-javascript.md

```diff
@@ -56,7 +56,7 @@ Sending an error message to the console.
 console.error("File not found!")
 ```
 
-Viewing the current stack trace.
+Viewing the current stack trace [1].
```

Comment:
```suggestion
Viewing the current stack trace[1].
```
If I am not mistaken, the footnotes `[` should be at the end of the word without a space

Reply:
True